debian/jitsi-meet-web-config.postinst

```diff
@@ -182,36 +186,57 @@ case "$1" in
             echo "config.flags.receiveMultipleVideoStreams = true;" >> $JITSI_MEET_CONFIG
         fi
 
-        if [[ "$FORCE_NGINX" = "true" && ( -z "$JVB_HOSTNAME_OLD" || "$RECONFIGURING" = "true" ) ]] ; then
+        NGINX_CONF_PATH="/etc/nginx/sites-enabled/$JVB_HOSTNAME_OLD.conf"
+        OPENRESTY_CONF_PATH="/etc/nginx/sites-enabled/$JVB_HOSTNAME_OLD.conf"
```

Comment:
What is the point of this? When it duplicates the value.
Can we simplify it?

Reply:
It is all right in cotext of right values of variables (I've already pushed the fix. Honestly, I don't know how on the earth I pushed this values, as I was pretty sure that I set proper values there :shrug: )